src/v0/destinations/sendinblue/util.js

```diff
@@ -0,0 +1,65 @@
+const { DESTINATION, EMAIL_SUFFIX } = require("./config");
+const { TransformationError } = require("../../util");
+const { TRANSFORMER_METRIC } = require("../../util/constant");
+
+const validateEmail = email => {
+  const regex = /^(([^\s"(),.:;<>@[\\\]]+(\.[^\s"(),.:;<>@[\\\]]+)*)|(".+"))@((\[(?:\d{1,3}\.){3}\d{1,3}])|(([\dA-Za-z-]+\.)+[A-Za-z]{2,}))$/;
+  return !!regex.test(email);
+};
+
+const validatePhoneWithCountryCode = phone => {
+  const regex = /^\+(?:[\d{] ?){6,14}\d$/;
+  return !!regex.test(phone);
+};
+
+const checkIfEmailAndPhoneExists = payload => {
+  if (!payload.email && !payload.phone) {
+    throw new TransformationError(
+      "At least one of email or phone is required",
+      400,
+      {
+        scope: TRANSFORMER_METRIC.MEASUREMENT_TYPE.TRANSFORMATION.SCOPE,
+        meta: TRANSFORMER_METRIC.MEASUREMENT_TYPE.TRANSFORMATION.META.BAD_EVENT
+      },
+      DESTINATION
+    );
+  }
+};
+
+const validateEmailAndPhone = payload => {
+  if (!validateEmail(payload.email)) {
+    throw new TransformationError(
+      "The provided email is invalid",
+      400,
+      {
+        scope: TRANSFORMER_METRIC.MEASUREMENT_TYPE.TRANSFORMATION.SCOPE,
+        meta: TRANSFORMER_METRIC.MEASUREMENT_TYPE.TRANSFORMATION.META.BAD_PARAM
+      },
+      DESTINATION
+    );
+  }
+
+  if (!validatePhoneWithCountryCode(payload.phone)) {
+    throw new TransformationError(
+      "The provided phone number is invalid",
+      400,
+      {
+        scope: TRANSFORMER_METRIC.MEASUREMENT_TYPE.TRANSFORMATION.SCOPE,
+        meta: TRANSFORMER_METRIC.MEASUREMENT_TYPE.TRANSFORMATION.META.BAD_PARAM
+      },
+      DESTINATION
+    );
+  }
+};
+
+const prepareEmailFromPhone = phone => {
```

Comment:
can we add some context about this? Do we have any public doc regarding this? If yes, lets link that. Or if it is totally experiment basis, add that detail in comments.

Reply:
This is experimental. Sendinblue is appending `@mailin-sms.com` to `phone` number and setting it in a `email` field if only phone is passed while creating a contact.

Reply:
then please add that in comments. Otherwise it will look confusing.

Reply:
comment added